.github/workflows/npm-publish.yml

```diff
@@ -7,16 +7,18 @@ on:
 
 jobs:
   publish:
-    # only allow on main or tags
-    if: ${{ github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/') }}
+    # only allow on main or tags on our repo
+    if: github.repository == 'dependents/node-detective-postcss' && (github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/'))
```

Comment:
Not sure about the extra parens, but `{{..}}` shouldn't be needed here. If this causes any issues, we can either remove the `github.repository` condition and the parens, or try to fix it later :)

Reply:
You can test it by tagging without publishing a GitHub release, that kicks the build off in dry run mode, so the package is not published. The current definition works, so we can also just keep it.

Reply:
If you revert this part, we can get the meaningful changes in separately from the cleanup.

Reply:
I can't actually test it because it's not on the main branch.

I'll revert the new condition addition, but we should probably try to add it later :)